tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleDependencies.java

```diff
@@ -487,9 +498,6 @@ public class ExampleDependencies {
                     .putAll(
                             "//ldml/dates/timeZoneNames/zone[@type=\"([^\"]*+)\"]/short/daylight",
                             "//ldml/personNames/personName[@order=\"([^\"]*+)\"][@length=\"([^\"]*+)\"][@usage=\"([^\"]*+)\"][@formality=\"([^\"]*+)\"]/namePattern")
-                    .putAll(
-                            "//ldml/dates/timeZoneNames/zone[@type=\"([^\"]*+)\"]/short/generic",
-                            "//ldml/personNames/personName[@order=\"([^\"]*+)\"][@length=\"([^\"]*+)\"][@usage=\"([^\"]*+)\"][@formality=\"([^\"]*+)\"]/namePattern")
```

Comment:
This is the only example dependency removed. I don't know why. The implication is that these namePattern paths no longer depend on these timeZoneNames paths.